index.html

```diff
@@ -2,13 +2,138 @@
+          <li>Теги для работы с формами.</li>
+          <li>Div-блоки.</li>
+        </ul>
+      </ol>
+    </b>
+    <table border="1" width="100%">
+      <thead>
+      <tr>
+        <th>Tag</th>
+        <th>Description</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr>
+        <td>&#60;table&#62;</td>
+        <td>Defines a table</td>
+      </tr>
+      <tr>
+        <td>&#60;tr&#62;</td>
+        <td>Defines a row in table</td>
+      </tr>
+      <tr>
+        <td>&#60;th&#62;</td>
+        <td rowspan="2">Defines a "header cell"/"cell" in table</td>
+      </tr>
+      <tr>
+        <td>&#60;td&#62;</td>
+      </tr>
+      <tr>
+        <td colspan="2">Rarely used</td>
+      </tr>
+      <tr>
+        <td>&#60;caption&#62;</td>
+        <td>Defines a table caption</td>
+      </tr>
+      <tr>
+        <td>&#60;thead&#62;</td>
+        <td>Groups the header content in a table</td>
+      </tr>
+      <tr>
+        <td>&#60;tbody&#62;</td>
+        <td>Groups the body content in a table</td>
+      </tr>
+      <tr>
+        <td>&#60;tfood&#62;</td>
+        <td>Groups the footer content in a table</td>
+      </tr>
+      </tbody>
+    </table>
+    <form>
+      <fieldset>
+        <legend>My first form</legend>
+        <p>
+          <label for="name">Name</label><br />
+          <input id="name" type="text" name="name" placeholder="Name" /><br>
+          <label for="email">Email</label><br />
+          <input id="email" type="email" name="email"/><br>
+          <label for="password">Password</label><br />
+          <input id="password" type="password" name="Password"/><br>
+          <label for="programminglevel">Your programming level</label><br />
+          <input id="programminglevel" type="range" name="programminglevel"/>
+        </p>
+        <p>
+          What does HTML stand for?<br>
+        </p>
+        <input id="answers-1" type="checkbox" name="question1" value="1">
+        <label for="answers-1">Hyperlinks and Text Markup Language</label><br />
+        <input id="answers-2" type="checkbox" name="question1" value="2">
+        <label for="answers-2">Home Tool Markup Language</label><br />
+        <input id="answers-3" type="checkbox" name="question1" value="3">
+        <label for="answers-3">Hyper Text Marcup Language</label>
+        <p>
+          What does HTTP stand for?
+        </p>
+        <input id="answer-1" type="radio" name="question" value="1">
+        <label for="answer-1">Hyper Text Translate Protokol</label><br />
+        <input id="answer-2" type="radio" name="question" value="2">
+        <label for="answer-2">Hyper Text Transmission Protocol</label><br />
+        <input id="answer-3" type="radio" name="question" value="3">
+        <label for="answer-3">Hyper Text Transfer Protocol</label><br />
+        <label for="comment">Comment</label><br>
+        <textarea id="comment" name="comment" placeholder="Comment"></textarea><br>
```

Comment:
The length of text doesn't match the task.

Reply:
I corrected.

Reply:
I have added the cols="22" attribute to the textarea. did I do right?

Reply:
Hmm, sure it is not limit the textarea content to 500 characters.